docs/tutorials/py-config-interpreter.md

```diff
@@ -75,8 +75,8 @@ To confirm that the interpreter is set correctly, you can open the DevTools and
   <body>
     <py-config>
       [[interpreters]]
-        src = "https://cdn.jsdelivr.net/pyodide/v0.22.0a3/full/pyodide.js"
-        name = "pyodide-0.22.0a3"
+        src = "https://cdn.jsdelivr.net/pyodide/v0.23.0/full/pyodide.js"
+        name = "pyodide-0.23.0"
```

Comment:
My thinking is that, since the purpose of this tutorial is setting a different version of the interpreter and being able to verify that it has happened, that this should actually not be the default version. I'll add a sentence in that section clarifying this.